front/pages/api/login.ts

```diff
@@ -66,275 +98,281 @@ async function handler(
     return;
   }
 
-  switch (req.method) {
-    case "GET":
-      // `workspaceInvite` is set to a `Workspace` if the query includes a `wId`. It means the user
-      // is going through the flow of whitelisted domain to join the workspace.
-      let workspaceInvite: null | Workspace = null;
-      let isAdminOnboarding = false;
-
-      if (req.query.wId) {
-        workspaceInvite = await Workspace.findOne({
-          where: {
-            sId: req.query.wId as string,
+  if (req.method !== "GET") {
```

Comment:
nit: it's done via a switch everywhere else across the code, maybe worth consitencying?

Reply:
Oh, I just saw you changed it from the switch.

I could agree it's better like this if pitched well, but even though, consistency with the rest of the codebase seems quite important
It's a nit, so feel free to merge with or without adressing

Reply:
also made the diff harder to read :sweat_smile:

Reply:
I understand the importance of code consistency, yet considering the length of our functions and that we're dealing with just a single scenario ("GET"), I believe removing the switch case here ease parsing this piece of code. It reduces indentation and allows for early returns. That being said, this does introduce a new pattern that does not exist in `handler` functions today. If you're ok with it, I'll keep the current approach and take a stab to refactor the other occurrences later on.

Reply:
yep :+1: